src/App.test.js

```diff
@@ -3,6 +3,6 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/Missions/i);
   expect(linkElement).toBeInTheDocument();
 });
```

Comment:
- [ ] Please add unit tests using the mentioned test libraries that you set up in your app 👍